packages/core/src/i18n.ts

```diff
@@ -173,31 +173,25 @@ export class I18n extends EventEmitter<Events> {
   }
 
   /**
-   * @param locales one locale or array of locales.
-   *   If array of locales is passed they would be used as fallback
-   *   locales for date and number formatting
+   * @param locale initial active locale.
+   * @param locales list of alternative locales (BCP 47 language tags) which are used for number and date formatting.
    * @param messages compiled message catalog
-   * @param notify Should emit `change` event for all subscribers.
-   *    This is useful for integration with frameworks as NextJS to avoid race-conditions during initialization.
    */
-  loadAndActivate(
-    locales: Locale | Locales,
```

Comment:
there's a reason why the existing `activate` call accepts `locale` and `locales` separately, because when you activate as in https://lingui.dev/ref/core#optionslocales 

then the `locale` will be used for translating strings and `locales` will be used for number formatting here https://github.com/lingui/js-lingui/blob/559a8c77811368176650e2773d2bec259d031033/packages/core/src/i18n.ts#L242

maintaining that behavior would not be possible with how `loadAndActivate` is done currently, so I had to refactor it